sdk/search/Azure.Search.Documents/src/Indexes/SearchIndexClient.cs

```diff
@@ -302,6 +302,54 @@ public virtual async Task<Response<SearchServiceStatistics>> GetServiceStatistic
                 throw;
             }
         }
+
+        /// <summary>
+        /// Retrieves a summary of statistics for all indexes in the search service.
+        /// </summary>
+        /// <param name="cancellationToken">Optional <see cref="CancellationToken"/> to propagate notifications that the operation should be canceled.</param>
+        /// <returns>The <see cref="Response{T}"/> from the server containing <see cref="ListIndexStatsSummary"/>.</returns>
+        /// <exception cref="RequestFailedException">Thrown when a failure is returned by the Search service.</exception>
+        public virtual Response<ListIndexStatsSummary> GetIndexStatsSummary(
```

Comment:
Can we add a test for this?

Reply:
Would love to, but unfortunately the endpoint is returning a 404 error at the moment. Alan checked with the service team and they expect it to be available early this week. We'll have it properly tested before we release it, for sure.

Reply:
Sounds good! Please create an issue to track this work item.

Reply:
Here it is: https://github.com/Azure/azure-sdk-for-net/issues/48721